src/Tribe/Aggregator/Cron.php

```diff
@@ -385,6 +386,75 @@ public function verify_child_record_creation() {
 		}
 	}
 
+	public function start_batch_pushing_records() {
+		if ( ! tribe( 'events-aggregator.main' )->is_service_active() ) {
+			return;
+		}
+
+		$records = Tribe__Events__Aggregator__Records::instance();
+
+		$query = $records->query( [
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 83b136eaf7930c6a57d5cbc31965d09a</sub>